qa/suites/deepsea/tier2/node-removal/2-test-phase.yaml

```diff
@@ -0,0 +1,9 @@
+tasks:
+        - deepsea.policy:
+                munge_policy:
+                        del_node:
+                                remove_me.test
```

Comment:
please change this line to `remove_me.test:` (and see below for resulting code simplification)